README.md

```diff
@@ -20,8 +20,8 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AndrewChew
 **Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AndrewChew/tree/master/core_java)]:
```

Comment:
**Linux Cluster Monitoring Agent** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AndrewChew/tree/master/linux_sql)]: Developed a Linux Cluster Monitoring Agent allowing users to obtain server hardware and usage statistics in real-time. Provisioned a PostgreSQL instance using a Docker container to persist data and designed a bash agent to be installed on servers to collect and insert data into the database. Implemented  SQL queries to answer business questions. (Perforemed DA using SQL to busniess/reports). Collaborated with the development team using the Agile workflow.